go.mod

```diff
@@ -3,9 +3,7 @@ module github.com/signalfx/signalfx-go-tracing
 go 1.12
```

Comment:
Shouldn't .circleci/go.mod and .circleci/go.sum be removed entirely and parent project's go.mod used instead or are doing this to install packages for all instrumentations at once? 

I think it would be better to use each module's own go.mod so local dev and CI are as close as possible. CI can then CD into each module and run `go test ./...` separately for each instrumentation package.

Reply:
They are moved to root folder 😉 See `.circleci/go.mod → go.mod `

Reply:
:) Thanks